build.gradle.kts

```diff
@@ -35,14 +33,11 @@ extensions.configure<ApiValidationExtension> {
 }
 
 allprojects {
-    repositories {
```

Comment:
I'd prefer to keep this here instead of moving it to `extra.gradle.kts`. It's cleaner to keep the list of repositories for plugins + libraries separate.

Reply:
`buildscript`'s `repositories` have been decleared separately, `extra.gradle.kts` provide common repos for `allprojects` and `buildSrc`

Reply:
`buildscript`'s `repositories` have been decleared separately, `extra.gradle.kts` provide common repos for `allprojects` and `buildSrc`

Reply:
![image](https://user-images.githubusercontent.com/10363352/105118851-1d0b1c00-5b0a-11eb-9f3b-ac582802c30b.png)